de-lib/src/main/java/org/iplantc/de/theme/base/client/admin/BelphegorDisplayStrings.properties

```diff
@@ -17,7 +17,7 @@ addCategoryPrompt = New Category
 appCategorizeSuccess = Successfully added {0} to the following Categories: {1,list}.
 appCategorizeSuccess[\=1] = Successfully added {0} to the following Category: {1,list}.
 confirmDeleteAppCategory = Do you want to delete the category \"{0}\"?
-confirmDeleteAppTitle = Delete App
+confirmDeprecateAppTitle = Delete App
```

Comment:
I was expecting the opposite change for this PR: internally we can still name methods and properties `*deleteApp*`, since we're still marking apps as deleted in our services, but we only need to change labels presented to the user from `delete` to `deprecate`.